services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import MovieSession
+
+from django.db.models import QuerySet
+
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time:
+        movie_session.show_time = show_time
+        movie_session.save()
+    if movie_id:
+        movie_session.movie_id = movie_id
+        movie_session.save()
+    if cinema_hall_id:
+        movie_session.cinema_hall_id = cinema_hall_id
+        movie_session.save()
```

Comment:
You should save movie_session once
```suggestion
    movie_session = MovieSession.objects.get(id=session_id)
    if show_time:
        movie_session.show_time = show_time
    if movie_id:
        movie_session.movie_id = movie_id
    if cinema_hall_id:
        movie_session.cinema_hall_id = cinema_hall_id
    movie_session.save()
```